Gemfile

```diff
@@ -2,11 +2,11 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'rake', if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.2")
-              '~> 13.0.1'
-            else
-              '< 13'
-            end
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.2")
```

Comment:
I think the next release could drop Ruby < 2.2

Reply:
So can we drop Ruby 2.1 from the CI matrix?
https://github.com/brianmario/mysql2/blob/80e0001f87ffd3974835478d2e9b068cc5d37f71/.github/workflows/build.yml#L18-L27

and also CentOS 7 with Ruby 2.0.0?
https://github.com/brianmario/mysql2/blob/80e0001f87ffd3974835478d2e9b068cc5d37f71/.github/workflows/container.yml#L14-L15